pyproject.toml

```diff
@@ -60,3 +60,6 @@ known-first-party = [
     "modal_version",
 ]
 extra-standard-library = ["pytest"]
+
+[tool.pyright]
+reportUnnecessaryComparison = true
```

Comment:
Found this gem for pyright that is deactivated by default - warns if a comparison would always evaluate to False because you mistyped a literal etc.